### PR TITLE
consume swagger changes for addparticipantresult

### PR DESCRIPTION
**Pull request by zihzhan-msft**

### CODE REVIEW

File: sdk/communication/azure-communication-callingserver/tests/recordings/test_live_call_connection.test_create_add_remove_hangup_scenario.yaml [hunk 1]
```diff
@@ -13,36 +13,36 @@ interactions:
...
-      - azsdk-python-communication-identity/1.0.1 Python/3.9.7 (Windows-10-10.0.19043-SP0)
+      - azsdk-python-communication-identity/1.1.0b2 Python/3.9.7 (Windows-10-10.0.22000-SP0)
       x-ms-date:
-      - Sat, 09 Oct 2021 21:59:09 GMT
+      - Sat, 04 Dec 2021 00:11:20 GMT
       x-ms-return-client-request-id:
       - 'true'
     method: POST
-    uri: https://sanitized.communication.azure.com/identities?api-version=2021-03-07
+    uri: https://sanitized.communication.azure.com/identities?api-version=2021-10-31-preview
```
Comment (satyakonmsft) [inline sdk/communication/azure-communication-callingserver/tests/recordings/test_live_call_connection.test_create_add_remove_hangup_scenario.yaml:22]: 2021-11-15-preview ?

File: sdk/communication/azure-communication-callingserver/tests/recordings/test_live_call_connection.test_create_add_remove_hangup_scenario.yaml [hunk 2]
```diff
@@ -13,36 +13,36 @@ interactions:
...
   response:
     body:
-      string: '{"identity":{"id":"8:acs:016a7064-0581-40b9-be73-6dde64d69d72_0000000d-06f2-cf20-c80d-454822000f38"}}'
+      string: '{"identity":{"id":"8:acs:016a7064-0581-40b9-be73-6dde64d69d72_0000000e-22a9-92dc-e3c7-593a0d0044f2"}}'
     headers:
       api-supported-versions:
       - 2020-07-20-preview2, 2021-02-22-preview1, 2021-03-07, 2021-03-31-preview1,
-        2021-10-31-preview
+        2021-10-31-preview, 2021-11-01
```
Comment (satyakonmsft) [inline sdk/communication/azure-communication-callingserver/tests/recordings/test_live_call_connection.test_create_add_remove_hangup_scenario.yaml:29]: why do u need to add this

**Review by satyakonmsft — APPROVED** [review]

File: sdk/communication/azure-communication-callingserver/tests/recordings/test_live_call_connection.test_create_add_remove_hangup_scenario.yaml (hunk 2 above)
Comment (RyanZzzz) [reply sdk/communication/azure-communication-callingserver/tests/recordings/test_live_call_connection.test_create_add_remove_hangup_scenario.yaml:29]: It's generated from the live test recording tool. Not  something we controlled.

Comment (zihzhan-msft) [reply sdk/communication/azure-communication-callingserver/tests/recordings/test_live_call_connection.test_create_add_remove_hangup_scenario.yaml:29]: It's generated from the live test recording tool. Not something we controlled.

**Pull request merged**